website/source/docs/providers/aws/r/volume_attachment.html.markdown

```diff
@@ -49,6 +49,10 @@ example, `/dev/sdh` or `xvdh`)
 volume to detach. Useful if previous attempts failed, but use this option only 
 as a last resort, as this can result in **data loss**. See 
 [Detaching an Amazon EBS Volume from an Instance][1] for more information.
+* `skip_destroy` - (Optional, Boolean) Set to `true` if you don't want to
+try and remove an EBS volume from a running disk. This is required in the
```

Comment:
"Set this to `true` if you do not wish to detach the volume from the instance to which it is attached at destroy time, and instead just remove the attachment from Terraform state. This is useful when destroying an instance which has volumes created by some other means attached."